torchchat/cli/cli.py

```diff
@@ -533,15 +533,16 @@ def arg_init(args):
         # Localized import to minimize expensive imports
         from torchchat.utils.build_utils import get_device_str
 
-        if args.device is None or args.device == "fast":
+        if args.device is None:
             args.device = get_device_str(
                 args.quantize.get("executor", {}).get("accelerator", default_device)
             )
         else:
+            args.device = get_device_str(args.device)
             executor_handler = args.quantize.get("executor", None)
             if executor_handler:
```

Comment:
plz merge the if and the following if into one line
```suggestion
            if executor_handler and executor_handler["accelerator"] != args.device:
```